boards/DIM/Test/Src/seven_seg_displays_test.cpp

```diff
@@ -0,0 +1,212 @@
+extern "C"
+{
+// The argument history needs to be long enough for us to test write every
+// one of the possible 16x16x16 = 4096 values on the three 7-segment displays
+#include <math.h>
+#include "App_SharedConstants.h"
+#define FFF_ARG_HISTORY_LEN (size_t) pow((float)NUM_HEX_DIGITS, (float)3)
+}
+
+#include "fff.h"
+#include "gtest/gtest.h"
+
+extern "C"
+{
+#include "App_SevenSegDisplays.h"
+#include "App_SevenSegDisplay.h"
+#include "App_ErrorCode.h"
+
+    DEFINE_FFF_GLOBALS;
+    FAKE_VOID_FUNC(set_right_hex_digit, struct SevenSegHexDigit);
```

Comment:
Wasn't sure if the argument should be a pointer or not. Maybe it doesn't matter too much because `struct SevenSegHexDigit` is quite small?